guides/v2.1/contributor-guide/backward-compatible-development/index.md

```diff
@@ -39,6 +39,12 @@ Mark the method with the `@deprecated` tag instead of removing it.
 
 Continue returning the same results from the method if possible, so the old functionality is preserved.
 
+If you need to remove the existing functionality and you do not need alternative implementation, you SHOULD change the method code to
```

Comment:
Also, "SHOULD" should be changed to "MUST". There are no exceptions to this rule.